projects/plugins/crm/admin/settings/custom-fields.page.php

```diff
@@ -301,8 +301,9 @@
 	 */
 	$sort_types_map = array(
 		'address'  => array(
-			'custom_key' => 'addresses',
-			'obj_key'    => 'zbsAddressFields',
+			'custom_key'   => 'addresses',
+			'obj_key'      => 'zbsAddressFields',
+			'force_prefix' => 'addr_',
```

Comment:
I think possibly naming 'force_prefix' to 'force_addr_prefix' might be more self explanatory?
```suggestion
			'force_addr_prefix' => 'addr_',
```

Then also affects line 341.

Reply:
Thanks! Addressed in a029281.